pnpm-workspace.yaml

```diff
@@ -3,7 +3,7 @@ packages:
   - 'apps/*'
 
 catalog:
-  expo: ~53.0.15
+  expo: 53.0.13
```

Comment:
@TimoGlastra if you want, I can test a bit more versions, but it depends a bit when we want to go to 54.